worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/ArgumentsManager.java

```diff
@@ -160,7 +174,12 @@ else if (matcher.group("type").equals("cd")){
 
         final ResolvedSetting resolvedSetting;
         try {
-            resolvedSetting = settingsApi.getResolvedSetting(name, String.join(",", scopes));
+            if(priorities.isEmpty()) {
+                resolvedSetting = settingsApi.getResolvedSetting(name, String.join(",", scopes), null);
```

Comment:
Surely the only way `priorities` is empty is if `scopes` is also empty, which makes the `String.join(",", scopes)` on this line strange.